### PR TITLE
Fix mongoDB database name references as dashboard in all modules.

### PR DESCRIPTION
**Pull request by gigaaks**
Hi
Please pull my changes are documentation for setting mongoDB database name is mentioned "dashboard" and "dashboard**db**" or "dashboard**b**" in different places (aka main or module level README.MD files). I fixed those so that other people won't face much issues.

Thanks.
Arun Sangal

### CODE REVIEW

**Comment by tabladrum** [comment]
@gigaaks Thanks for the PR. Did you submit the CLA?

**Comment by gigaaks** [comment]
No. How should I. I didn't check if there are references of dashboard, dashboardb, dashboarddb (for values) of mongoDB database names in the actual java/configuration files. Just changed the README.md files. I'll check those too.

**Comment by tabladrum** [comment]
https://docs.google.com/forms/d/19LpBBjykHPox18vrZvBbZUcK6gQTj7qv1O5hCduAZFU/viewform

**Comment by gigaaks** [comment]
Hm, for some reason I can't open that link. Let me try again from a different device.

**Comment by gigaaks** [comment]
Ok, CLA is complete at my end.

**Comment by gigaaks** [comment]
Pls advise if anything else is required at my end. Thanks a lot.

**Comment by tabladrum** [comment]
Thanks for the PR again. I re-opened it so that I can do the merge.

**Comment by gigaaks** [comment]
I'm looking forward if either myself or someone can contribute to writing collectors for XLRelease, CA LARA and CA Rally tools as well.

**Comment by tabladrum** [comment]
@gigaaks XLRelease etc collectors would be awesome.
